lib/_http_outgoing.js

```diff
@@ -262,15 +264,14 @@ function _writeRaw(data, encoding, callback) {
       this._flushOutput(conn);
     } else if (!data.length) {
       if (typeof callback === 'function') {
-        let socketAsyncId = this.socket[async_id_symbol];
         // If the socket was set directly it won't be correctly initialized
         // with an async_id_symbol.
         // TODO(AndreasMadsen): @trevnorris suggested some more correct
         // solutions in:
         // https://github.com/nodejs/node/pull/14389/files#r128522202
-        if (socketAsyncId === undefined) socketAsyncId = null;
-
-        nextTick(socketAsyncId, callback);
+        defaultTriggerAsyncIdScope(conn[async_id_symbol],
```

Comment:
Is `conn[async_id_symbol] === this.socket[async_id_symbol]`? Also, this changes the `socketAsyncId` so if this is correct the comment should be updated too.

Reply:
`conn === this.socket` so it's equivalent. Not quite clear on the second part of the feedback — the comment above it still applies.